mkch/cron.sls

```diff
@@ -4,3 +4,13 @@
     - contents: |
         #!/bin/sh
         zypper -n up -l --no-recommends
+
+/etc/cron.hourly/update-automation:
+  file.managed:
+    - mode: 0755
+    - contents: |
+        #!/bin/sh
+        # managed by salt
+        if [ -x /root/bin/update_automation ]; then
+            /root/bin/update_automation
```

Comment:
please use the default checkout path `~/github.com/SUSE-Cloud/automation/scripts/update-automation` and not the deprecated symlinks

Reply:
That path doesn't exist on either mkchp, or in my checkout of automation master.

Reply:
You should be able to use `~/github.com/SUSE-Cloud/automation/scripts/jenkins/update_automation`